### DISCUSSION TITLE
An example for positive composite model

### DISCUSSION THREAD

**Discussion by akhilmabraham** [pybamm-team/PyBaMM#3759] [Q&A]
The composite model notebook below explains the script for Si-Gr blend. 
https://docs.pybamm.org/en/stable/source/examples/notebooks/models/composite_particle.html

The notebook says it can be extended to positive composite electrodes and a reference is attached. I don't see an instruction or reference on how it's extended to positive electrode. Can @rtimms @DrSOKane @TomTranter @brosaplanella @tinosulzer  please help with or comment to this discussion how it's implemented for positive electrode. I am familiar with pybamm modelling, I can share my jupyter notebook on positive composite electrode after that.

**Comment by TomTranter** [comment]
`{"particle phases": ("2", "1")}` refers to there being 2 phases in the anode and 1 in the cathode. I think you would need to change this and then provide the relevant secondary phase parameters

**Reply by akhilmabraham** [reply]
Thanks @TomTranter. I am struggling with the latter. How do we define or input parameter value for secondary phase?

**Reply by rtimms** [reply]
Yes, you will need to pass the option `{"particle phases": ("1", "2")}`. In the parameters you need to prepend "Primary: " and "Secondary: " to the phase parameters, see e.g. [here](https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/input/parameters/lithium_ion/Chen2020_composite.py).
